src/lib/ecs/system.ts

```diff
@@ -475,7 +503,7 @@ export class SystemManager {
 			system.dt = os.clock() - system.lastCalled;
 			system.lastCalled = os.clock();
 
-			debug.profilebegin("system: " + system.name);
+			debug.profilebegin("system: " + tostring(getmetatable(system)));
```

Comment:
I don't like how our performance enforcer contains a concatenation 🤔
As long as people name their systems well, simply using the system name should be sufficient.